app/core/src/main/java/com/fsck/k9/K9.kt

```diff
@@ -227,7 +227,7 @@ object K9 : EarlyInit {
     var isUseVolumeKeysForListNavigation = false
 
     @JvmStatic
-    var isHideSpecialAccounts = false
+    var isHideUnifiedInbox = false
```

Comment:
If we're renaming this anyway, let's make it `isShowUnifiedInbox`. If find that makes for more readable code.

Reply:
done